cypress/e2e/error_spec.cy.js

```diff
@@ -0,0 +1,16 @@
+describe('error page', () => {
+  it('should display an error page if failed to get fact', () => {
+    cy.intercept('https://catfact.ninja/fact', {
+      statusCode: 500
+    })
+    cy.visit('http://localhost:3000/')
+    cy.get('button[class=fact-btn]').click()
```

Comment:
It seems like the user needs to click on a button first before seeing the error message.  Is this what you wanted?

Reply:
My thought was that was the only way the error would occur since there is no fetch calls being made on page load. @

Reply:
Got it, makes sense!